site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -275,8 +276,10 @@ That means that you skipped the `--cleanup` flag and you should either delete th
 
 ## Controller TLS Certificates
 
-By default agones chart generates tls certificates used by the admission controller, while this is handy, it requires the agones controller to restart on each `helm upgrade` command.
-For most use cases the controller would have required a restart anyway (eg: controller image updated). However if you really need to avoid restarts we suggest that you turn off tls automatic generation (`agones.controller.generateTLS` to `false`) and provide your own certificates (`certs/server.crt`,`certs/server.key`).
+By default agones chart generates tls certificates used by the admission controller and reuses them on each `helm upgrade` command using `lookup` templating function.
```

Comment:
Please wrap this in a feature shortcode (see https://agones.dev/site/docs/contribute/#within-a-page) so that the old text is still published until the next release, when this behavior becomes the new default.

Reply:
done